docs/05-technical-reference/svls-06-function-configuration-file.md

```diff
@@ -8,6 +8,12 @@ When you initialize a Function (with the `init` command), CLI creates the `confi
 
 See the sample `config.yaml` for an inline Function for which code and dependencies are stored in the Function custom resource (CR) under the **spec.source** and **spec.deps** fields. This specification also contains the definition of a sample Subscription and APIRules for the Function:
 
+<div tabs name="specification" group="function-configuration-file">
+  <details>
+  <summary label="v1alpha1">
+  v1alpha1
```

Comment:
this field should contain `v1` instead of `v1alpha1` it's the specification version, not the resource version

```suggestion
  <summary label="v0">
  v0
```